src/lib/requestToken/index.ts

```
@@ -179,6 +228,21 @@ let signerFailureDetail = "";
  */
 let initInFlight: Promise<void> | undefined;
 
+/**
+ * The session the layer was last given a code for. Kept because the code
+ * itself goes into the signer and is not stored anywhere else — on the
+ * failure path there is no signer, so without this there would be nothing
+ * to re-initialise with.
+ */
+let lastSession:
+  | { sessionCode: string; sessionCodeExpiresAt: number }
+  | undefined;
```

ESLint

[error] Replace `|·{·sessionCode:·string;·sessionCodeExpiresAt:·number·}⏎·` with `{·sessionCode:·string;·sessionCodeExpiresAt:·number·}`

(prettier/prettier)